src/mewbot/api/registry.py

```diff
@@ -132,6 +134,12 @@ def register_api_version(
         """
 
         def do_register(api: type[Component]) -> type[Component]:
+            """
+            See the doc string of the function which returns this function.
```

Comment:
Feels that

> See :mewbot.api.registry:register_api_version

would be more user friendly. (but in whatever the sphix syntax is.